addon_service/configured_addon/link/views.py

```diff
@@ -12,3 +21,19 @@ class ConfiguredLinkAddonViewSet(ConfiguredAddonViewSet):
         "authorized_resource",
     )
     serializer_class = ConfiguredLinkAddonSerializer
+
+    @action(
+        detail=True,
+        methods=[HTTPMethod.GET],
+        url_name="verified-link",
+        url_path="verified-link",
+    )
+    def verified_link(self, request, pk=None):
+        addons: ConfiguredLinkAddon = ConfiguredLinkAddon.objects.filter(
+            authorized_resource__resource_uri__in=[
+                f"{prefix}/{pk}" for prefix in ALLOWED_RESOURCE_URI_PREFIXES
+            ],
+        ).select_related("base_account__external_service")
+        self.resource_name = "verified-link"
+
+        return Response(VerifiedLink(addons, many=True).data)
```

Comment:
Ah, I just learned this one today: https://www.django-rest-framework.org/api-guide/serializers/#dealing-with-multiple-objects